settings/src/main/scala/org/typelevel/sbt/TypelevelSettingsPlugin.scala

```diff
@@ -201,7 +205,29 @@ object TypelevelSettingsPlugin extends AutoPlugin {
         Seq("-Werror")
       else
         Seq.empty
+    },
+    scalacOptions ++= {
+      val releaseOption = if (isJava8) Seq() else Seq("-release", tlJvmRelease.value.toString)
+      val targetOption = if (isJava8) Seq() else Seq(s"-target:${tlJvmRelease.value}")
+
+      scalaVersion.value match {
```

Comment:
I know it's ancient but Scala 2.11 supports this too.
https://github.com/typelevel/cats-parse/blob/7b524daf2abe63ca896656cfd36c13e9051476f6/build.sbt#L87

Reply:
(I wouldn't ask if there weren't some TL projects still on 2.11).

Reply:
Done. It's a bit opinionated as nothing before JDK 8 is supported by `sbt-typelevel`. I hope that's ok.

Reply:
Yeah, I think that's more than reasonable.